operatorapi/volumes.go

```diff
@@ -230,43 +231,61 @@ func getPVCEventsResponse(session *models.Principal, params operator_api.GetPVCE
 	return retval, nil
 }
 
-func getTenantCSResponse(session *models.Principal, params operator_api.ListTenantCertificateSigningRequestParams) (*models.CsrElement, *models.Error) {
+func getTenantCSResponse(session *models.Principal, params operator_api.ListTenantCertificateSigningRequestParams) (*models.CsrElements, *models.Error) {
 	ctx, cancel := context.WithCancel(params.HTTPRequest.Context())
 	defer cancel()
 	clientset, err := cluster.K8sClient(session.STSSessionToken)
 	if err != nil {
 		return nil, errors.ErrorWithContext(ctx, err)
 	}
-	csrName := params.Tenant + "-" + params.Namespace + "-csr"
-	csrResult, csrError := clientset.CertificatesV1().CertificateSigningRequests().Get(ctx, csrName, metav1.GetOptions{})
-	if csrError != nil {
-		return nil, errors.ErrorWithContext(ctx, err)
-	}
-	annotations := []*models.Annotation{}
-	for k, v := range csrResult.ObjectMeta.Annotations {
-		annotations = append(annotations, &models.Annotation{Key: k, Value: v})
-	}
-	var DeletionGracePeriodSeconds int64
-	DeletionGracePeriodSeconds = 0
-	if csrResult.ObjectMeta.DeletionGracePeriodSeconds != nil {
-		DeletionGracePeriodSeconds = *csrResult.ObjectMeta.DeletionGracePeriodSeconds
+
+	// Get CSRs by Label "v1.min.io/tenant=" + params.Tenant
+	listByTenantLabel := metav1.ListOptions{LabelSelector: "v1.min.io/tenant=" + params.Tenant}
+	listResult, _ := clientset.CertificatesV1().CertificateSigningRequests().List(ctx, listByTenantLabel)
```

Comment:
```suggestion
	listResult, err := clientset.CertificatesV1().CertificateSigningRequests().List(ctx, listByTenantLabel)
```
Please handle this errors here, a nil `listResult` may end on a `NPE` because of `listResult.Items`